### PR TITLE
Reenable auto checks on PRs

### PR DESCRIPTION
**Pull request by theopnv**
### Purpose of this PR

- Added back the pipelines that were removed to alleviate the load on Yamato back in April.
The pipelines were removed on April 28th so I browsed a snapshot of the repo from just before (`4a23bfca06c6b533131042e410ea0fef0ba06503`) and moved the jobs as they were at that time.

---
### Testing status
Tested the triggers by editing a few hdrp files (see last commits). Then reverted the changes and canceled the pipelines.

### CODE REVIEW

File: .yamato/all-hdrp.yml [hunk 1]
```diff
@@ -20,8 +20,26 @@ PR_HDRP_trunk:
...
+         rerun: on_new_revision
+      -  path: .yamato/hdrp-osx-metal.yml#HDRP_OSX_Metal_playmode_mono_Linear_trunk
+         rerun: on_new_revision
+      -  path: .yamato/hdrp-linux-vulkan.yml#HDRP_Linux_Vulkan_playmode_mono_Linear_trunk
+         rerun: on_new_revision
+      -  path: .yamato/hdrp-linux-vulkan.yml#HDRP_Linux_Vulkan_Standalone_mono_Linear_trunk
+         rerun: on_new_revision
       -  path: .yamato/all-hdrp_dxr.yml#PR_HDRP_DXR_trunk
          rerun: on_new_revision
+      -  path: .yamato/all-hdrp_hybrid.yml#PR_HDRP_Hybrid_trunk
```
Comment (kivistikliis) [inline .yamato/all-hdrp.yml:41]: Should Hybrids be excluded still?

File: .yamato/all-smoke_test.yml [hunk 2]
```diff
@@ -30,10 +34,6 @@ Extra_Smoke_Test_trunk:
     dependencies:
       -  path: .yamato/all-smoke_test.yml#PR_Smoke_Test_trunk
```
Comment (kivistikliis) [inline .yamato/all-smoke_test.yml:35]: Is the dependency on the PR job intended? (I think this was from an earlier PR, not this one, but wondering it nevertheless)

File: .yamato/all-urp_2d.yml [hunk 3]
```diff
@@ -60,6 +46,22 @@ Extra_URP_2D_trunk:
...
          rerun: on_new_revision
+Extra_URP_2D_trunk:
+    name: Extra URP_2D on trunk
+    variables:
+        CUSTOM_REVISION: '{{trunk.changeset.id}}'
+        UTR_VERSION: "current"
+        TEST_FILTER: .*
+        CACHE_ARGS: '{{cache.flags}}'
+    dependencies:
+      -  path: .yamato/urp_2d-win-dx11.yml#URP_2D_Win_DX11_playmode_mono_Linear_trunk
```
Comment (kivistikliis) [inline .yamato/all-urp_2d.yml:57]: This one seems to be also included in the PR job

File: .yamato/all-urp_lighting.yml [hunk 4]
```diff
@@ -16,24 +16,6 @@ PR_URP_Lighting_trunk:
         TEST_FILTER: .*
         CACHE_ARGS: '{{cache.flags}}'
     dependencies:
-      -  path: .yamato/urp_lighting-win-dx11.yml#URP_Lighting_Win_DX11_Standalone_mono_Linear_trunk
```
Comment (kivistikliis) [inline .yamato/all-urp_lighting.yml:19]: Did this job get removed from the PR job?

File: .yamato/all-urp_postpro.yml [hunk 5]
```diff
@@ -16,24 +16,6 @@ PR_URP_PostPro_trunk:
         TEST_FILTER: .*
         CACHE_ARGS: '{{cache.flags}}'
     dependencies:
-      -  path: .yamato/urp_postpro-win-dx11.yml#URP_PostPro_Win_DX11_Standalone_mono_Linear_trunk
```
Comment (kivistikliis) [inline .yamato/all-urp_postpro.yml:19]: Same here

File: .yamato/all-urp_terrain.yml [hunk 6]
```diff
@@ -16,24 +16,6 @@ PR_URP_Terrain_trunk:
         TEST_FILTER: .*
         CACHE_ARGS: '{{cache.flags}}'
     dependencies:
-      -  path: .yamato/urp_terrain-win-dx11.yml#URP_Terrain_Win_DX11_Standalone_mono_Linear_trunk
```
Comment (kivistikliis) [inline .yamato/all-urp_terrain.yml:19]: Same here

File: .yamato/all-hdrp.yml (hunk 1 above)
Comment (theo-pnv) [reply .yamato/all-hdrp.yml:41]: Good question. I tried finding where the Nightly Hybrid was called back in April https://github.com/Unity-Technologies/Graphics/tree/4a23bfca06c6b533131042e410ea0fef0ba06503/.yamato, but I can't find any pipeline calling it, or trigger.
Aren't we supposed to run some hybrid tests, at least in the nightly?

**Review by kivistikliis — APPROVED** [review]

**Pull request merged**